fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/flaps_computer.rs

```diff
@@ -265,15 +270,16 @@ impl SlatFlapControlComputer {
         &mut self,
         context: &UpdateContext,
         adirs: &impl AdirsMeasurementOutputs,
-        flaps_handle: &CSUMonitor,
         flaps_feedback: &impl PositionPickoffUnit,
         slats_feedback: &impl PositionPickoffUnit,
     ) {
-        self.flaps_handle_position = flaps_handle.get_current_detent();
-        self.flaps_conf = self.generate_configuration(flaps_handle, context, adirs);
-        self.flap_load_relief_active = self.flap_load_relief_active(flaps_handle);
-        self.cruise_baulk_active = self.cruise_baulk_active(flaps_handle);
-        self.alpha_speed_lock_active = self.alpha_speed_lock_active(flaps_handle);
+        self.csu_monitor.update(context);
+
+        self.flaps_handle_position = self.csu_monitor.get_current_detent();
+        self.flaps_conf = self.generate_configuration(&self.csu_monitor, context, adirs);
+        self.flap_load_relief_active = self.flap_load_relief_active(&self.csu_monitor);
+        self.cruise_baulk_active = self.cruise_baulk_active(&self.csu_monitor);
+        self.alpha_speed_lock_active = self.alpha_speed_lock_active(&self.csu_monitor);
```

Comment:
All these functions can now access `csu_monitor` themself so no need to pass it as an argument.